crates/aptos/src/common/types.rs

```diff
@@ -78,6 +78,14 @@ const ACCEPTED_CLOCK_SKEW_US: u64 = 5 * US_IN_SECS;
 pub const DEFAULT_EXPIRATION_SECS: u64 = 30;
 pub const DEFAULT_PROFILE: &str = "default";
 pub const GIT_IGNORE: &str = ".gitignore";
+#[cfg(windows)]
```

Comment:
Consider using [indoc](https://github.com/dtolnay/indoc), it'd make this a lot more readable. Or use a vec of strings.